pages/settings/preferences/index.vue

```diff
@@ -15,162 +15,172 @@ const userSettings = useUserSettings()
-    >
-      {{ $t('settings.preferences.hide_reply_count') }}
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'hideFollowerCount')"
-      @click="togglePreferences('hideFollowerCount')"
-    >
-      {{ $t('settings.preferences.hide_follower_count') }}
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'hideUsernameEmojis')"
-      @click="togglePreferences('hideUsernameEmojis')"
-    >
-      {{ $t("settings.preferences.hide_username_emojis") }}
-      <template #description>
-        {{ $t('settings.preferences.hide_username_emojis_description') }}
-      </template>
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'hideNews')"
-      @click="togglePreferences('hideNews')"
-    >
-      {{ $t("settings.preferences.hide_news") }}
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'zenMode')"
-      @click="togglePreferences('zenMode')"
-    >
-      {{ $t("settings.preferences.zen_mode") }}
-      <template #description>
-        {{ $t('settings.preferences.zen_mode_description') }}
-      </template>
-    </SettingsToggleItem>
-    <h2 px6 py4 mt2 font-bold text-xl flex="~ gap-1" items-center>
-      <div i-ri-flask-line />
-      {{ $t('settings.preferences.title') }}
-    </h2>
-    <!-- Embedded Media -->
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'experimentalEmbeddedMedia')"
-      @click="togglePreferences('experimentalEmbeddedMedia')"
-    >
-      {{ $t('settings.preferences.embedded_media') }}
-      <template #description>
-        {{ $t('settings.preferences.embedded_media_description') }}
-      </template>
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'experimentalVirtualScroller')"
-      @click="togglePreferences('experimentalVirtualScroller')"
-    >
-      {{ $t('settings.preferences.virtual_scroll') }}
-      <template #description>
-        {{ $t('settings.preferences.virtual_scroll_description') }}
-      </template>
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'experimentalGitHubCards')"
-      @click="togglePreferences('experimentalGitHubCards')"
-    >
-      {{ $t('settings.preferences.github_cards') }}
-      <template #description>
-        {{ $t('settings.preferences.github_cards_description') }}
-      </template>
-    </SettingsToggleItem>
-    <SettingsToggleItem
-      :checked="getPreferences(userSettings, 'experimentalUserPicker')"
-      @click="togglePreferences('experimentalUserPicker')"
-    >
-      {{ $t('settings.preferences.user_picker') }}
-      <template #description>
-        {{ $t('settings.preferences.user_picker_description') }}
-      </template>
-    </SettingsToggleItem>
+    <section>
+      <h2 px6 py4 mt2 font-bold text-xl flex="~ gap-1" items-center sr-only>
```

Comment:
rn using sr-only here, maybe we should add a new i18n entry `General Preferences` : if sr-only is ok then we should remove the icon

Reply:
Maybe adding `General Preferences` is more consistent so that every section can have a header. VoiceOver behavior sounds good so either is fine for now.